src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -322,6 +324,8 @@ class ResourceService(
       validateRoles(resourceType, policy) ++
       validateUrlSafe(policy.policyName.value)
 
+    // todo: validate descendant permissions??
```

Comment:
As far as I can tell, I do need to add something here to make sure that the policy isn't inherited by any other resources and to throw a 400 if it is